### ISSUE TITLE
Scattergl symbols + 100 are not empty

### ISSUE THREAD

**Issue by nvaytet** [plotly/plotly.py#1928]
When trying to plot empty symbols with `Scattergl` by adding 100 to a symbol number, the symbols are not empty (while the marker actually appears empty in the legend).

Using `Scatter` instead of `Scattergl` works as expected.

Finally, adding `"-open"` to the symbol name does work with `Scattergl`.

Code example:
```Py
import plotly.graph_objects as go
import numpy as np

N = 10
fig = go.Figure()
fig.add_trace(go.Scattergl(x=np.arange(N),
                           y=np.random.random(N),
                           mode='markers',
                           marker=dict(symbol=101, size=20)))
fig.add_trace(go.Scattergl(x=np.arange(N),
                           y=np.random.random(N),
                           mode='markers',
                           marker=dict(symbol="square-open", size=20)))
fig.add_trace(go.Scatter(x=np.arange(N),
                           y=np.random.random(N),
                           mode='markers',
                           marker=dict(symbol=101, size=20)))
```

yields

![Screenshot at 2019-11-25 22-05-43](https://user-images.githubusercontent.com/39047984/69578113-13525b00-0fd0-11ea-8972-c510c509a072.png)

**Comment by nicolaskruchten** [comment]
Thanks for the bug report! We'll fix this in https://github.com/plotly/plotly.js/issues/2507 and the fix should come out in version 4.4

**Comment by nicolaskruchten** [comment]
This works in `master` and will definitely be released in 4.5.0 :)
